src/Apps/Legion/Components/StepperWizard/StepperWizard.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import StepperWizard from './StepperWizard';
+import {
+    IStepperWizardProps,
+    IStepperWizardStep,
+    StepperWizardType
+} from './StepperWizard.types';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import { WizardNavigationContextProvider } from '../../Models/Context/WizardNavigationContext/WizardNavigationContext';
+
+const wrapperStyle = { width: 'fit-content', height: 'auto' };
+
+export default {
+    title: 'Components/Apps/Legion/StepperWizard',
```

Comment:
Can we group all those under WizardShell story?
![image](https://user-images.githubusercontent.com/45217314/228630933-b3ce6987-c686-4bff-9067-9f2fdbac64ae.png)
```suggestion
    title: 'Components/Apps/Legion/WizardShell/StepperWizard',
```